### ISSUE TITLE
Russian characters

### ISSUE THREAD

**Issue by hram** [openstf/stf#701]
stf version 3.1.0

Hi!
Can you help me.
How can I type russian characters from PC keyboard?
English characters from PC keyboard and russian keyboard on device works fine.

**Comment by sorccu** [comment]
I don't think that works. From what I know only Latin languages and some languages with a Latin-based IME (e.g. Japanese) work quite well.

It's not ideal, but you can tap on the software keys on the device itself.
